.github/workflows/haskell.yml

```diff
@@ -288,6 +291,8 @@ jobs:
     # - name: Setup tmate session
     #   if: ${{ failure() }}
     #   uses: mxschmitt/action-tmate@v3
+    #   with:
+    #     limit-access-to-actor: true
```

Comment:
Adding this to the commented out `tmate` step for security reasons in case someone decides to use it.